kernel/Base/MeshManipulator_Impl.h

```diff
@@ -1601,7 +1659,7 @@ void MeshManipulator<DIM>::readMesh(const std::string &filename) {
         input >> partition;
         Base::Element *element = nullptr;
         if (partition == processorID) {
-            element = addElement(coordinateIndices, partition, true);
+            element = addElement(coordinateIndices, nullptr, partition, true);
```

Comment:
I would give the nullptr a name,
```suggestion
*zone default=nullptr;
bool owning=true;
            element = addElement(coordinateIndices, default, partition, owning);
```